onadata/apps/main/management/commands/create_enketo_express_urls.py

```diff
@@ -38,7 +38,11 @@ def handle(self, *args, **options):
             raise CommandError(
                 'please provide a server_name, a server_port and a protocol')
 
-        if server_name not in ['api.ona.io', 'stage-api.ona.io', 'localhost']:
+        if server_name not in [
+                'api.ona.io',
+                'stage-api.ona.io',
+                'localhost',
+                'api.preview.ona.io']:
```

Comment:
I think it would be better if we introduced a setting like `APP_DOMAIN` that has the value that should be used in this function... I don't believe we ever need to create enketo express urls for other domains other than the one the application is running on... thought ?

Reply:
Would also ensure that this command is able to run on any domain

Reply:
cool cool....sounds cleaner